cmd/neofs-cli/modules/bearer/create.go

```diff
@@ -35,14 +34,14 @@ All epoch flags can be specified relative to the current epoch with the +n synta
 In this case --` + commonflags.RPC + ` flag should be specified and the epoch in bearer token
 is set to current epoch + n.
 `,
-	RunE: createToken,
+	Run: createToken,
 }
 
 func init() {
 	createCmd.Flags().StringP(eaclFlag, "e", "", "path to the extended ACL table")
 	createCmd.Flags().StringP(issuedAtFlag, "i", "", "epoch to issue token at")
 	createCmd.Flags().StringP(notValidBeforeFlag, "n", "", "not valid before epoch")
-	createCmd.Flags().StringP(expireAtFlag, "x", "", "expiration epoch")
+	createCmd.Flags().StringP(commonflags.ExpireAt, "x", "", "expiration epoch")
```

Comment:
somewhere shorthand is `x` and somewhere is `e`, lets's also sync it

Reply:
It is left so deliberatly, `e` is for `eacl` here, I think equal long names is enough.